### DISCUSSION TITLE
SQLite change column

### DISCUSSION THREAD

**Discussion by gnimor** [dbeaver/dbeaver#11299] [Q&A] [answered]
I am trying to modify an existing column in a SQLite database (e.g.  set the "Auto Increment" flag). The database contains no data. Anyway the "Auto Increment", "Not Null" options are all grayed out. In addition I am not able to define a new key.

![DBeaver](https://user-images.githubusercontent.com/14901306/107626631-614b9100-6c5e-11eb-8779-2c18e855920b.png)

**Comment by uslss** [comment, marked as answer]
Unfortunately, mentioned features are not supported by SQLite
https://www.sqlitetutorial.net/sqlite-alter-table/
